src/main/resources/application.properties

```diff
@@ -59,3 +59,7 @@ spring.mail.properties.mail.smtp.auth: false
 #spring.mail.properties.mail.smtp.starttls.enable: false
 
 er.url: http://localhost:8080/search/
+
+# Please enable this property in test environment only, This /rules/rule-check/aggregation endpoint should work in test environment only
+# In production environment remove this property or false the value
+testaggregated.enabled: true
```

Comment:
this should be false even in the pull request so that you do not accidentally merge with value true